pkg/route53/route53.go

```diff
@@ -107,3 +109,61 @@ func (r *Route53) DeleteHostedZoneRecordSets(hostedZoneID string) (*route.Change
 
 	return resp, nil
 }
+
+// SetNSRecordTTL finds the NS record in a hosted zone, and sets the TTL for it. If there are zere or more than one
+// NS records, an error is thrown.
+func (r *Route53) SetNSRecordTTL(hostedZoneID string, nsTTLSecs int64) error {
+	ListResourceRecordSetsInput := route.ListResourceRecordSetsInput{
+		HostedZoneId: aws.String(hostedZoneID),
+	}
+
+	got, err := r.provider.Route53().ListResourceRecordSets(&ListResourceRecordSetsInput)
+	if err != nil {
+		return err
+	}
+
+	sets := got.ResourceRecordSets
+	nsCount := 0
+
+	var nsSet *route.ResourceRecordSet
+
+	for _, set := range sets {
+		if *set.Type == "NS" {
+			nsCount++
+
+			nsSet = set
+		}
+	}
+
+	if nsCount != 1 {
+		return errors.Errorf("expected 1 NS records in hosted zone, but found %d", nsCount)
```

Comment:
Don't we have 4 NS records in a hosted zone? ... Or is it maybe 4 values in the one records, yes, never mind ;)

Reply:
Yes, the last one :100: